mod/bookmarks/elgg-plugin.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+return [
+	'actions' => [
+		'logged_in' => [
+			'bookmarks/save' => 'bookmarks/save.php',
+			'bookmarks/delete' => 'bookmarks/delete.php',
+		],
+	],
+	'views' => [
+		'default' => [
+			'bookmarks/' => __DIR__ . '/graphics',
```

Comment:
Unless we plan to fully parse into a tree, a string format like `"{{DIR}}/graphics"` would be easiest to work with include/var_export.